src/components/drag-and-drop/draggable-context/draggable-context.js

```diff
@@ -152,14 +239,22 @@ class DraggableContext extends React.Component {
   handleEndDrag = () => {
     // dragging has ended so remove the active index
     this.setState({ activeIndex: null });
+    this.speed = 0;
+  }
+
+  handleMouseMove = (ev) => {
+    if (this.props.autoScroll && this.state.activeIndex !== null) this.checkAutoScrollTrigger(ev);
   }
 
   /**
    * Renders the component
    */
   render() {
     return (
-      <div className='carbon-draggable-context'>
+      <div
+        className='carbon-draggable-context'
+        onMouseMove={ this.handleMouseMove }
```

Comment:
can this listener be assigned depending on props.autoScroll and state.activeIndex, rather than the callback handling it?